User-Service/Data/Data Access/UserService.cs

```
@@ -0,0 +1,35 @@
+﻿using User_Service.Data;
+using User_Service.Model;
+
+namespace User_Service
+{
+    public class UserService
+    {
+        private readonly AppDbContext context;
+        public UserService(AppDbContext context)
+        {
+            this.context = context;
+        }
+      public String GetUserGoogleIdbyUserID(Guid userId)
+
+        {
+            // Returns the googleID of the user by the gived Guid userID
+
+            try
+            {
+
+                string googleID = context.Users.Where(x => x.UserId == userId).Select(x => x.GoogleId).ToString();
+                return googleID;
+
+            }
+            catch (InvalidCastException e)
```

This will only return a "InvalidCastException", either catch all exceptions. This should also rather be handled by exception middleware to keep code cleaner